projects/logging/application.ts

```diff
@@ -1,18 +1,15 @@
 import express = require('express')
 import { Request, Response } from 'express'
 import { logger } from './logger'
-import { LogFormat, Level } from '../Apps/common/src/logging'
+import { MallardLogFormat } from '../Apps/common/src/logging'
 
-const processLog = (rawData: LogFormat[]) => {
+const processLog = (rawData: MallardLogFormat[]) => {
     rawData.forEach(logData => {
-        logger.info({
-            '@timestamp': logData['@timestamp']
-                ? new Date(logData['@timestamp'])
-                : new Date(),
-            level: logData.level || Level.INFO,
-            message: logData.message || '',
```

Comment:
Did you want to keep this structure post processing?

Reply:
I decided I don't care if level isn't provided as I think the ELK thing  will just default to info, so the only thing needed here is to convert the timestamp and check that message exists